src/sentry/api/endpoints/organization_config_integrations.py

```diff
@@ -18,4 +18,8 @@ def get(self, request, organization):
             providers, organization=organization, serializer=IntegrationProviderSerializer()
         )
 
+        if "provider_key" in request.GET:
+            serialized = list(filter(lambda d: d["key"] in request.GET["provider_key"], serialized))
+            return Response({"information": serialized[0]})
```

Comment:
@NisanthanNanthakumar we shouldn't change the shape of the response with a query parameter like this. See `organization_integrations.py`

Reply:
We should also add a unit test for this change on the backend